### PR TITLE
feat: Created test files for book, patron, library and an exception

### PR DESCRIPTION
**Pull request by NatayaP**
feat: Created test files for book, patron, library and an exception

### CODE REVIEW

**Comment by NatayaP** [comment]
feat: Created book, patron, and library files

File: lesson_09/oop/oop_app/src/main/java/com/codedifferently/lesson9/natayaprice/Book.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
+package com.codedifferently.lesson9.natayaprice;
+
+public class Book {
+
+  private String Titles;
```
Comment (anthonydmays) [inline lesson_09/oop/oop_app/src/main/java/com/codedifferently/lesson9/natayaprice/Book.java:5]: Class members and methods should be `camelCase`, here and below.

File: lesson_09/oop/oop_app/src/main/java/com/codedifferently/lesson9/natayaprice/Book.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    return Titles;
+  }
+
+  /**
+   * Gets the ISBN number of the book.
+   *
+   * @return the ISBN number of the book
+   */
+  public int getISBN() {
+    return ISBN;
```
Comment (anthonydmays) [inline lesson_09/oop/oop_app/src/main/java/com/codedifferently/lesson9/natayaprice/Book.java:43]: Should be `isbn`.

File: lesson_09/oop/oop_app/src/main/java/com/codedifferently/lesson9/natayaprice/Library.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
+package com.codedifferently.lesson9.natayaprice;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Library {
```
Comment (anthonydmays) [inline lesson_09/oop/oop_app/src/main/java/com/codedifferently/lesson9/natayaprice/Library.java:6]: Should also support removing books, in addition to the check in/check out flow.

File: lesson_09/oop/oop_app/src/main/java/com/codedifferently/lesson9/natayaprice/Patron.java [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  public Patron(String string, String string2) {
+    // TODO Auto-generated constructor stub
+  }
+
+  /**
+   * The main method to execute tests for Patron functionalities.
+   *
+   * @param args the command line arguments
+   */
+  public static void main(String[] args) {
```
Comment (anthonydmays) [inline lesson_09/oop/oop_app/src/main/java/com/codedifferently/lesson9/natayaprice/Patron.java:23]: Nope, shouldn't have this and I don't think this runs anyway.

File: lesson_09/oop/oop_app/src/main/java/com/codedifferently/lesson9/natayaprice/Patron.java [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+public class Patron {
+
+  /**
+   * Constructs a new Patron with the specified name and email.
+   *
+   * @param name the name of the patron
+   * @param email the email of the patron
+   */
+  public Patron(String string, String string2) {
```
Comment (anthonydmays) [inline lesson_09/oop/oop_app/src/main/java/com/codedifferently/lesson9/natayaprice/Patron.java:14]: Should remove.

**Review by anthonydmays — APPROVED** [review]

**Pull request merged**